{"name": "test_calc_1[5-5-9]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 == 9", "trace": "input1 = 5, input2 = 5, output = 9\n\n    @ pytest.mark.parametrize(\"input1, input2, output\",\n                              [\n                                  (2,3,5),\n                                  (3,3,6),\n                                  (5,5,9)\n                              ]\n                              )\n    def test_calc_1(input1, input2, output):\n        result = sum(input1,input2)\n>       assert result == output\nE       assert 10 == 9\n\nTest_parameterization_demo.py:13: AssertionError"}, "parameters": [{"name": "input1", "value": "5"}, {"name": "input2", "value": "5"}, {"name": "output", "value": "9"}], "start": 1575221758599, "stop": 1575221758599, "uuid": "9f043ee7-da77-4be5-bc94-58dd82ee64f0", "historyId": "a746a1245bd5baa059a65854fb49f4ce", "testCaseId": "5a181d683c3740351bdf2541f2217bf0", "fullName": "test_parameterization_demo#test_calc_1", "labels": [{"name": "suite", "value": "test_parameterization_demo"}, {"name": "host", "value": "iMac.local"}, {"name": "thread", "value": "66278-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_parameterization_demo"}]}